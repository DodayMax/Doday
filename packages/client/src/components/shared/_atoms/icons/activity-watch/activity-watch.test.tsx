import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('ActivityWatchType', () => {
  it('renders correctly without props', () => {
    const icon = shallow(<Icons.ActivityWatchType />);

    expect(icon).toMatchSnapshot();
  });
});
