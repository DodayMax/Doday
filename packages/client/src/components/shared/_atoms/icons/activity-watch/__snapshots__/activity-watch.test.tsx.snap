// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityWatchType renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <g
        fill="black"
        fillRule="evenodd"
      >
        <path
          d="M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z"
        />
        <path
          d="M10.578 9.377l4.093 2.572-4.093 2.571V9.377zm0-1.543c-.848 0-1.535.691-1.535 1.543v5.143c0 .56.304 1.077.792 1.348a1.528 1.528 0 0 0 1.556-.042l4.093-2.571a1.545 1.545 0 0 0 0-2.618l-4.093-2.571a1.53 1.53 0 0 0-.813-.237v.005z"
        />
      </g>,
      "height": 20,
      "viewBox": "0 0 24 24",
      "width": 20,
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <path
            d="M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z"
          />,
          <path
            d="M10.578 9.377l4.093 2.572-4.093 2.571V9.377zm0-1.543c-.848 0-1.535.691-1.535 1.543v5.143c0 .56.304 1.077.792 1.348a1.528 1.528 0 0 0 1.556-.042l4.093-2.571a1.545 1.545 0 0 0 0-2.618l-4.093-2.571a1.53 1.53 0 0 0-.813-.237v.005z"
          />,
        ],
        "fill": "black",
        "fillRule": "evenodd",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M10.578 9.377l4.093 2.572-4.093 2.571V9.377zm0-1.543c-.848 0-1.535.691-1.535 1.543v5.143c0 .56.304 1.077.792 1.348a1.528 1.528 0 0 0 1.556-.042l4.093-2.571a1.545 1.545 0 0 0 0-2.618l-4.093-2.571a1.53 1.53 0 0 0-.813-.237v.005z",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
      ],
      "type": "g",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <g
          fill="black"
          fillRule="evenodd"
        >
          <path
            d="M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z"
          />
          <path
            d="M10.578 9.377l4.093 2.572-4.093 2.571V9.377zm0-1.543c-.848 0-1.535.691-1.535 1.543v5.143c0 .56.304 1.077.792 1.348a1.528 1.528 0 0 0 1.556-.042l4.093-2.571a1.545 1.545 0 0 0 0-2.618l-4.093-2.571a1.53 1.53 0 0 0-.813-.237v.005z"
          />
        </g>,
        "height": 20,
        "viewBox": "0 0 24 24",
        "width": 20,
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <path
              d="M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z"
            />,
            <path
              d="M10.578 9.377l4.093 2.572-4.093 2.571V9.377zm0-1.543c-.848 0-1.535.691-1.535 1.543v5.143c0 .56.304 1.077.792 1.348a1.528 1.528 0 0 0 1.556-.042l4.093-2.571a1.545 1.545 0 0 0 0-2.618l-4.093-2.571a1.53 1.53 0 0 0-.813-.237v.005z"
            />,
          ],
          "fill": "black",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M17.71 4.749a3.622 3.622 0 0 1 3.612 3.63v7.144a3.622 3.622 0 0 1-3.612 3.63H6.515a3.622 3.622 0 0 1-3.612-3.63V8.379a3.622 3.622 0 0 1 3.612-3.63h11.2-.005zm.005-1.543h-11.2c-2.841.003-5.144 2.317-5.147 5.173v7.144c.006 2.854 2.308 5.166 5.147 5.168h11.2c2.841-.002 5.144-2.317 5.147-5.173V8.374c-.006-2.856-2.31-5.168-5.152-5.168h.005z",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M10.578 9.377l4.093 2.572-4.093 2.571V9.377zm0-1.543c-.848 0-1.535.691-1.535 1.543v5.143c0 .56.304 1.077.792 1.348a1.528 1.528 0 0 0 1.556-.042l4.093-2.571a1.545 1.545 0 0 0 0-2.618l-4.093-2.571a1.53 1.53 0 0 0-.813-.237v.005z",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
        ],
        "type": "g",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
